common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -443,6 +443,18 @@ protected long delayNanos(long currentTimeNanos) {
         return scheduledTask.delayNanos(currentTimeNanos);
     }
 
+    /**
+     * Returns the absolute point in time (relative to {@link #nanoTime()}) at which the the next
+     * closest scheduled task should run.
+     */
+    protected long deadlineNanos() {
```

Comment:
I may remove this to address https://github.com/netty/netty/issues/7829 ... so lets hold of on releasing this.

Reply:
@normanmaurer ^